### PR TITLE
Update livepeer

### PR DESCRIPTION
**Pull request by bb-face**

### CODE REVIEW

File: gateway/src/App.js [hunk 1]
```diff
@@ -7,149 +7,120 @@ import "react-bootstrap-typeahead/css/Typeahead.bs5.css";
...
-  createReactClient,
-  studioProvider,
-} from "@livepeer/react";
 import { useAccount, useInitNear } from "near-social-vm";
 import {
-  Link,
-  Route,
-  BrowserRouter as Router,
-  useLocation,
+	Link,
```
Comment (elliotBraem) [inline gateway/src/App.js:12]: Can you add a prettier to this repo? Let's use this one: https://github.com/PotLock/bos-app/blob/main/.prettierrc

File: gateway/src/components/Broadcast.js [hunk 2]
```diff
@@ -1,13 +1,95 @@
...
+import { EnableVideoIcon, StopIcon } from "@livepeer/react/assets";
+import * as Broadcast from "@livepeer/react/broadcast";
+import { getIngest } from "@livepeer/react/external";
+
+export const BroadcastComponent = (props) => {
+	const [streamKey, setStreamKey] = useState(null);
+
+	const createStream = async () => {
+		try {
+			const response = await fetch("https://livepeer.studio/api/stream", {
```
Comment (elliotBraem) [inline gateway/src/components/Broadcast.js:11]: thoughts on moving this to a file w/ api's?

File: gateway/src/components/Player.js [hunk 3]
```diff
@@ -1,13 +1,63 @@
...
+	const src = getSrc(playbackInfo.playbackInfo);
+
+	return src;
+};
+
+export const VideoPlayer = (props) => {
+	const [src, setSrc] = useState(null);
+	const [playbackId, setPlaybackId] = useState(0);
+
+	useEffect(() => {
```
Comment (elliotBraem) [inline gateway/src/components/Player.js:22]: What was the challenge you were facing to have these playbackId useEffects?

File: gateway/src/components/Broadcast.js (hunk 2 above)
Comment (elliotBraem) [reply gateway/src/components/Broadcast.js:11]: After looking through rest of files, maybe it doesn't matter so much. Although, what do you think when we're planning for this next task? 

https://github.com/NEARBuilders/trylivepeer.near/issues/4

File: biome.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (elliotBraem) [inline biome.json:1]: Oh -- I take back prettier. What's biome? And any reason you prefer one over the other?

Comment (bb-face) [reply biome.json:1]: biome is prettier built in rust: https://biomejs.dev/
I find it much quicker, the problem in this project is that being an unfamiliar pattern sometimes doesn't really know what to do, like when you have a return outside of a function!

File: gateway/src/components/Broadcast.js (hunk 2 above)
Comment (bb-face) [reply gateway/src/components/Broadcast.js:11]: It's definitely something that I was thinking about. Separate the components in api (for fetching), create (create a stream for Broadcast or create a playbackId for a video) and displaying (the Player and Broadcast components)

File: apps/trylivepeer.near/widget/Player.jsx [hunk 5]
```diff
@@ -3,6 +3,6 @@
 return (
   <>
     {/* need to be explicit with props when passing from widget to VM */}
-    <Player value={props.key} />
+    <Player value={props.key} props={props.playerProps}/>
```
Comment (elliotBraem) [inline apps/trylivepeer.near/widget/Player.jsx:6]: I'm not loving the `props.props.playbackId` I see in components/Player...

Maybe we could be more explicit here like <Player playbackId={...} />

Also, value={props.key} was just an example, you can remove this

Comment (bb-face) [reply apps/trylivepeer.near/widget/Player.jsx:6]: Definitely, I don't love it either, it was something WIP!

**Pull request merged**